src/App.tsx

```diff
@@ -21,14 +60,15 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
+                ? <TodoList todos={filteredTodos} />
+                : <Loader />}
             </div>
           </div>
         </div>
       </div>
 
-      <TodoModal />
+      {selectedTodo && (<TodoModal />)}
```

Comment:
```suggestion
      {!!selectedTodo && (<TodoModal />)}
```
in rendering with && for best rendering control use !! to return a boolean value